build-plugin/action.yml

```diff
@@ -0,0 +1,63 @@
+name: Build Plugin
+description: Inject the given version into the plugin files and then build it and zip it.
+
+inputs:
+  PACKAGE_VERSION:
+    required: true
+    description: 'The version to inject into the plugin package.json file.'
+  REPO_NAME:
+    required: true
+    description: 'The name of the repository to build the plugin for.'
+
+
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      run: |
+            if [ "${{ inputs.PACKAGE_VERSION }}" == "" ] || [ "${{ inputs.PACKAGE_VERSION }}" == "undefined" ];
```

Comment:
Make it generic, move all dynamic values to be configured by the `inputs`

Reply:
done